# UpdatePose.srv
# This service is designed to specify localization
# transformation manually
geometry_msgs/TransformStamped transformation
---
